sjb/config/test_cases/test_branch_origin_extended_conformance_install_update.yml

```diff
@@ -23,6 +23,10 @@ extensions:
       script: |-
         tito_tmp_dir="tito"
         mkdir -p "${tito_tmp_dir}"
+        if [[ ${ORIGIN_TARGET_BRANCH} == release-1.* ]]
+        then
+          git checkout -b ${ORIGIN_TARGET_BRANCH} origin/${ORIGIN_TARGET_BRANCH}
```

Comment:
I dont think so, cause when you check the origin repo branches you can see what branches are there, in this case only `release-1.x` and `stage-1.x` ... and those reflect to the o-a branches... `enterprise-1.x` branches are only present in OCP